docs/advanced/addons/managed-dhcp.md

```diff
@@ -0,0 +1,254 @@
+---
+sidebar_position: 6
+sidebar_label: Managed DHCP
+title: "Managed DHCP"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/advanced/managed-dhcp"/>
+</head>
+
+_Available as of v1.3.0_
+
+Beginning with v1.3.0, you can configure IP pool information and serve IP addresses to VMs running on Harvester clusters using the embedded Managed DHCP feature. This feature, which is an alternative to the standalone DHCP server, leverages the vm-dhcp-controller add-on to simplify cluster deployment.
+
+:::note
+Harvester uses the planned infrastructure network so you must ensure that network connectivity is available and plan the IP pools in advance.
+:::
+
+This is accomplished using the vm-dhcp-controller add-on.
```

Comment:
> This is accomplished using the vm-dhcp-controller add-on.

Please remove this. I added the information to the first paragraph already.

Reply:
Updated. Thank you!